### PR TITLE
Add gitignore for GoogleService-Info.plist

### PR DESCRIPTION
**Pull request by jeremyjiang-dev**

### CODE REVIEW

**Review by paulb777 — COMMENTED** [review]
Please update the top level `.gitignore` instead with `FirebasePerformance/**/GoogleService-Info.plist` instead.

**Comment by jeremyjiang-dev** [comment]
> Please update the top level `.gitignore` instead with `FirebasePerformance/**/GoogleService-Info.plist` instead.

Thanks Paul! Updated.

**Review by paulb777 — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -35,6 +35,9 @@ FirebaseSegmentation/Tests/Sample/GoogleService-Info.plist
 # FirebaseMLModelDownloader integration tests GoogleService-Info.plist
 FirebaseMLModelDownloader/Tests/Integration/Resources/GoogleService-Info.plist
 
+# FirebasePerformance dev test App and integration tests GoogleService-Info.plist
+FirebasePerformance/**/GoogleService-Info.plist
```
Comment (zijianjoy) [inline .gitignore:39]: Should we ignore this plist file in the whole repo instead of just FirebasePerformance?

**Review by zijianjoy — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (visumickey) [reply .gitignore:39]: +1 @paulb777 What do you think?

**Review by visumickey — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (paulb777) [reply .gitignore:39]: No. We have some dummy plists in the repo that are sufficient for build only and some unit testing.

Comment (jeremyjiang-dev) [reply .gitignore:39]: Make sense. I will ignore the plist only in FirebasePerformance.

**Pull request merged**